src/date_de_serialization.rs

```diff
@@ -0,0 +1,31 @@
+use anyhow::Result;
+use chrono::prelude::*;
+use serde::de;
+use std::fmt;
+
+struct DateTimeVisitor;
+
+impl<'de> de::Visitor<'de> for DateTimeVisitor {
+    type Value = DateTime<Utc>;
+
+    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        write!(formatter, "a string represents chrono::DateTime<Utc>")
+    }
+
+    fn visit_str<E>(self, s: &str) -> Result<Self::Value, E>
+    where
+        E: de::Error,
+    {
+        match Utc.datetime_from_str(s, "%d/%m/%Y %H:%M:%S") {
+            Ok(t) => Ok(t),
+            Err(_) => Err(de::Error::invalid_value(de::Unexpected::Str(s), &self)),
+        }
+    }
+}
+
+pub fn from_date<'de, D>(d: D) -> Result<DateTime<Utc>, D::Error>
```

Comment:
It looks like the `chrono` crate [has a `serde` feature](https://github.com/chronotope/chrono/blob/3467172c31188006147585f6ed3727629d642fed/Cargo.toml#L43) which might add a `Deserialize` implementation for the `DateTime` type. From their README:

> - `serde`: Enable serialization/deserialization via serde.